lib/rules/jsx-boolean-value.js

```diff
@@ -106,16 +123,21 @@ module.exports = {
           additionalProperties: false,
           properties: {
             [ALWAYS]: exceptionsSchema,
+            assumeUndefinedIsFalse: {
+              type: 'boolean',
+            },
           },
-        }],
+        },
+        ],
         additionalItems: false,
-      }],
+      },
+      ],
```

Comment:
making these changes would force a lot more whitespace diffs; please revert them (if you have autoformatting enabled in your editor, please disable it; eslint is the only autoformatter that should be used in this project)

Reply:
Yes I fixed it by changing the IDE settings.